docs/development.md

```diff
@@ -1,28 +1,22 @@
 # Development
 
-Add details here to give a brief overview of how to work with the provider APIs.
-Please reference any SDKs or API docs used to help build the integration here.
-
-## Prerequisites
-
-Supply details about software or tooling (like maybe Docker or Terraform) that
-is needed for development here.
-
-Please supply references to documentation that details how to install those
-dependencies here.
-
-Tools like Node.js and NPM are already covered in the [README](../README.md) so
-don't bother documenting that here.
+This integration uses Veracode's
+[findings api](https://docs.veracode.com/r/c_findings_v2_intro)
 
 ## Provider account setup
 
-Please provide information about the steps needed to create an account with a
-provider. Images and references to a provider's documentation is very helpful
-for new developers picking up your work.
+See the `In Veracode` (TODO: link to jupiterone.md) section of the official
```

Comment:
I'm guessing this `TODO` is for after the first merge to `main`?

Reply:
yup 👍